src/nupic/database/ClientJobsDAO.py

```diff
@@ -55,8 +55,7 @@ def _getLogger():
   needs to be initialized by the host app, and that usually happens after
   imports
   """
-  return logging.getLogger(
-    ".".join(['com.numenta', _MODULE_NAME, ClientJobsDAO.__name__]))
+  return logging.getLogger(__name__)
```

Comment:
Actually I think it would be better to make a module-level constant so it doesn't keep getting re-created.